src/mudlet-lua/lua/utf8_filenames.lua

```diff
@@ -164,9 +165,17 @@ local function modify_lua_functions(all_compressed_mappings)
       print"-------------------------------------------------"
    end
 
-   if getOS() == "windows" then
+   if (os.getenv"os" or ""):match"^Windows" then
 
-      local codepage = getWindowsCodepage()
```

Comment:
I would also keep this change because it uses a built-in Mudlet function, instead of opening a `cmd` window which is annoying for the user.

After this change is reverted... what exactly is this improvement bringing?

Reply:
Well it is accessing the appropriate key in the registry which is the *right-thing* on Windows for getting settings... 

but looking into our function I see it is doing the same thing so ours is better - but it does mean that we have forked from upstream. **However we are still using elements of that which does not have a GPL-like license so we still need to include the license for the original work...**